src/components/ShowPost.tsx

```diff
@@ -1,34 +1,46 @@
-import React, { Component } from 'react';
+import * as React from 'react';
+import { useEffect } from 'react';
+
 import { connect } from 'react-redux';
 import { requestFetch, requestDelete } from '../actions';
 
-class ShowPost extends Component {
-  componentDidMount() {
-    this.props.requestFetch();
-  }
+interface ShowPostProps {
+  text: string[];
+  postText: { fetchPostData: { postText: string[] }; key: string[] };
+  fetchPostData: string;
```

Comment:
fetchPostData, postText is verb, so these should be function, but these are Object and string. so please fix name

Reply:
maybe, postText should be `data[]`.

nits: please rename `data`, because this is __very__ unkind name and unreadable code

Reply:
まず、postText:  fetchPostDataを、text,fetchDateに変更すればいいでしょうか？

Reply:
`fetchData` has yet to be noun.

and text is __very__ unkind.

You should name each variable, function, or property in a way that is easy to understand so that everyone can understand what it means at a glance

Reply:
If you want to know how important naming is, please read this article 

https://megalodon.jp/2019-0105-0145-49/note.mu/kotofurumiya/n/n31d401fce782